tests/machine.test.js

```diff
@@ -201,6 +201,20 @@ describe(OlmMachine.name, () => {
         expect(await machine().displayName).toBeUndefined();
     });
 
+    test("can toggle room key requests", async () => {
+        const m = await machine();
+        expect(m.roomKeyRequestsEnabled).toBeTruthy();
```

Comment:
Curious, why do we not use true and false in these tests? We're not dealing with null, undefined et al., so a triple-equal comparison would return what we expect.